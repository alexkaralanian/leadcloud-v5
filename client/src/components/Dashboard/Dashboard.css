.dashHeader {
  font-size: 2.5em;
  color: #4a90e2;
  font-weight: 100;
  margin-bottom: 20px;
  text-align: center;
}

.dashCard {
  background-color: #4a90e2;
  padding: 10px;
  margin: 10px 0;
  border: 1px solid grey;
  border-radius: 2px;
}

.dashText {
  font-weight: 100;
  font-size: 2em;
  color: white;
  font-family: "Helvetica Neue";
}

.navLink {
  text-decoration: none !important;
}

.content {
  padding-top: 10px;
}
