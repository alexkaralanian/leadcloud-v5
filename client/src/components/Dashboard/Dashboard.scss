.dashboard {
  &__content {
    &-card {
      background-color: #4a90e2;
      padding: 1rem;
      margin-bottom: 1rem;
      border: 1px solid grey;
      border-radius: 2px;
    }

    &-text {
      font-weight: 100;
      font-size: 2.5rem;
      color: white;
      font-family: "Helvetica Neue";
    }

    &-link {
      text-decoration: none !important;
    }
  }
}
