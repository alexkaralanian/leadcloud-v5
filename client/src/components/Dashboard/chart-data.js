const brandPrimary = "#20a8d8";
const brandSuccess = "#4dbd74";
const brandInfo = "#63c2de";
const brandWarning = "#f8cb00";
const brandDanger = "#f86c6b";

// Card Chart 1
export const cardChartData1 = {
  labels: ["January", "February", "March", "April", "May", "June", "July"],
  datasets: [
    {
      label: "My First dataset",
      backgroundColor: brandPrimary,
      borderColor: "rgba(255,255,255,.55)",
      data: [65, 59, 84, 84, 51, 55, 40]
    }
  ]
};

export const cardChartOpts1 = {
  maintainAspectRatio: false,
  legend: {
    display: false
  },
  scales: {
    xAxes: [
      {
        gridLines: {
          color: "transparent",
          zeroLineColor: "transparent"
        },
        ticks: {
          fontSize: 2,
          fontColor: "transparent"
        }
      }
    ],
    yAxes: [
      {
        display: false,
        ticks: {
          display: false,
          min: Math.min.apply(Math, cardChartData1.datasets[0].data) - 5,
          max: Math.max.apply(Math, cardChartData1.datasets[0].data) + 5
        }
      }
    ]
  },
  elements: {
    line: {
      borderWidth: 1
    },
    point: {
      radius: 4,
      hitRadius: 10,
      hoverRadius: 4
    }
  }
};

// Card Chart 2
export const cardChartData2 = {
  labels: ["January", "February", "March", "April", "May", "June", "July"],
  datasets: [
    {
      label: "My First dataset",
      backgroundColor: brandInfo,
      borderColor: "rgba(255,255,255,.55)",
      data: [1, 18, 9, 17, 34, 22, 11]
    }
  ]
};

export const cardChartOpts2 = {
  maintainAspectRatio: false,
  legend: {
    display: false
  },
  scales: {
    xAxes: [
      {
        gridLines: {
          color: "transparent",
          zeroLineColor: "transparent"
        },
        ticks: {
          fontSize: 2,
          fontColor: "transparent"
        }
      }
    ],
    yAxes: [
      {
        display: false,
        ticks: {
          display: false,
          min: Math.min.apply(Math, cardChartData2.datasets[0].data) - 5,
          max: Math.max.apply(Math, cardChartData2.datasets[0].data) + 5
        }
      }
    ]
  },
  elements: {
    line: {
      tension: 0.00001,
      borderWidth: 1
    },
    point: {
      radius: 4,
      hitRadius: 10,
      hoverRadius: 4
    }
  }
};
