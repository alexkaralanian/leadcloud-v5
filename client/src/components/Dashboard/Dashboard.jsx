import React from "react";
import { Link } from "react-router-dom";
import PropTypes from "prop-types";
import { Col, Row } from "reactstrap";
import "./Dashboard.scss";

const Dashboard = () => (
  <Row>
    <Col sm="6">
      <Link className="dashboard__content-link" to="/contacts">
        <div className="dashboard__content-card">
          <h3 className="dashboard__content-text">Contacts</h3>
        </div>
      </Link>
    </Col>
    <Col sm="6">
      <Link className="dashboard__content-link" to="/listings">
        <div className="dashboard__content-card">
          <h3 className="dashboard__content-text">Listings</h3>
        </div>
      </Link>
    </Col>
    <Col sm="6">
      <Link className="dashboard__content-link" to="/groups">
        <div className="dashboard__content-card">
          <h3 className="dashboard__content-text">Groups</h3>
        </div>
      </Link>
    </Col>
    <Col sm="6">
      <Link className="dashboard__content-link" to="/campaigns">
        <div className="dashboard__content-card">
          <h3 className="dashboard__content-text">Campaigns</h3>
        </div>
      </Link>
    </Col>
    <Col sm="6">
      <Link className="dashboard__content-link" to="/emails">
        <div className="dashboard__content-card">
          <h3 className="dashboard__content-text">Messages</h3>
        </div>
      </Link>
    </Col>
  </Row>
);

export default Dashboard;
