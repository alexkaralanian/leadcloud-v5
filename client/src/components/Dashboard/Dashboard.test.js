import React from "react";
import { shallow, configure } from "enzyme";
import Dashboard from "./Dashboard";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

test("Dashboard Component renders correctly", () => {
  const component = shallow(<Dashboard />);
  expect(component).toMatchSnapshot();
});
