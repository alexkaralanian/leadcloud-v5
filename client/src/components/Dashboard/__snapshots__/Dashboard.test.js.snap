// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Dashboard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Grid
        bsClass="container"
        className="content"
        componentClass="div"
        fluid={false}
      >
        <Row
          bsClass="row"
          componentClass="div"
        >
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/contacts"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Contacts
                </h3>
              </div>
            </Link>
          </Col>
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/listings"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Listings
                </h3>
              </div>
            </Link>
          </Col>
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/emails"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Emails
                </h3>
              </div>
            </Link>
          </Col>
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/profile"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Profile
                </h3>
              </div>
            </Link>
          </Col>
        </Row>
      </Grid>,
      "className": "background",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsClass": "container",
        "children": <Row
          bsClass="row"
          componentClass="div"
        >
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/contacts"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Contacts
                </h3>
              </div>
            </Link>
          </Col>
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/listings"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Listings
                </h3>
              </div>
            </Link>
          </Col>
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/emails"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Emails
                </h3>
              </div>
            </Link>
          </Col>
          <Col
            bsClass="col"
            componentClass="div"
            sm={6}
          >
            <Link
              className="navLink"
              replace={false}
              to="/profile"
            >
              <div
                className="dashCard"
              >
                <h3
                  className="dashText"
                >
                  Profile
                </h3>
              </div>
            </Link>
          </Col>
        </Row>,
        "className": "content",
        "componentClass": "div",
        "fluid": false,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "row",
          "children": Array [
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/contacts"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Contacts
                  </h3>
                </div>
              </Link>
            </Col>,
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/listings"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Listings
                  </h3>
                </div>
              </Link>
            </Col>,
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/emails"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Emails
                  </h3>
                </div>
              </Link>
            </Col>,
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/profile"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Profile
                  </h3>
                </div>
              </Link>
            </Col>,
          ],
          "componentClass": "div",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "col",
              "children": <Link
                className="navLink"
                replace={false}
                to="/contacts"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Contacts
                  </h3>
                </div>
              </Link>,
              "componentClass": "div",
              "sm": 6,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Contacts
                  </h3>
                </div>,
                "className": "navLink",
                "replace": false,
                "to": "/contacts",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h3
                    className="dashText"
                  >
                    Contacts
                  </h3>,
                  "className": "dashCard",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Contacts",
                    "className": "dashText",
                  },
                  "ref": null,
                  "rendered": "Contacts",
                  "type": "h3",
                },
                "type": "div",
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "col",
              "children": <Link
                className="navLink"
                replace={false}
                to="/listings"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Listings
                  </h3>
                </div>
              </Link>,
              "componentClass": "div",
              "sm": 6,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Listings
                  </h3>
                </div>,
                "className": "navLink",
                "replace": false,
                "to": "/listings",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h3
                    className="dashText"
                  >
                    Listings
                  </h3>,
                  "className": "dashCard",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Listings",
                    "className": "dashText",
                  },
                  "ref": null,
                  "rendered": "Listings",
                  "type": "h3",
                },
                "type": "div",
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "col",
              "children": <Link
                className="navLink"
                replace={false}
                to="/emails"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Emails
                  </h3>
                </div>
              </Link>,
              "componentClass": "div",
              "sm": 6,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Emails
                  </h3>
                </div>,
                "className": "navLink",
                "replace": false,
                "to": "/emails",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h3
                    className="dashText"
                  >
                    Emails
                  </h3>,
                  "className": "dashCard",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Emails",
                    "className": "dashText",
                  },
                  "ref": null,
                  "rendered": "Emails",
                  "type": "h3",
                },
                "type": "div",
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "col",
              "children": <Link
                className="navLink"
                replace={false}
                to="/profile"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Profile
                  </h3>
                </div>
              </Link>,
              "componentClass": "div",
              "sm": 6,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Profile
                  </h3>
                </div>,
                "className": "navLink",
                "replace": false,
                "to": "/profile",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h3
                    className="dashText"
                  >
                    Profile
                  </h3>,
                  "className": "dashCard",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Profile",
                    "className": "dashText",
                  },
                  "ref": null,
                  "rendered": "Profile",
                  "type": "h3",
                },
                "type": "div",
              },
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Grid
          bsClass="container"
          className="content"
          componentClass="div"
          fluid={false}
        >
          <Row
            bsClass="row"
            componentClass="div"
          >
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/contacts"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Contacts
                  </h3>
                </div>
              </Link>
            </Col>
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/listings"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Listings
                  </h3>
                </div>
              </Link>
            </Col>
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/emails"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Emails
                  </h3>
                </div>
              </Link>
            </Col>
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/profile"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Profile
                  </h3>
                </div>
              </Link>
            </Col>
          </Row>
        </Grid>,
        "className": "background",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "container",
          "children": <Row
            bsClass="row"
            componentClass="div"
          >
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/contacts"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Contacts
                  </h3>
                </div>
              </Link>
            </Col>
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/listings"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Listings
                  </h3>
                </div>
              </Link>
            </Col>
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/emails"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Emails
                  </h3>
                </div>
              </Link>
            </Col>
            <Col
              bsClass="col"
              componentClass="div"
              sm={6}
            >
              <Link
                className="navLink"
                replace={false}
                to="/profile"
              >
                <div
                  className="dashCard"
                >
                  <h3
                    className="dashText"
                  >
                    Profile
                  </h3>
                </div>
              </Link>
            </Col>
          </Row>,
          "className": "content",
          "componentClass": "div",
          "fluid": false,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "row",
            "children": Array [
              <Col
                bsClass="col"
                componentClass="div"
                sm={6}
              >
                <Link
                  className="navLink"
                  replace={false}
                  to="/contacts"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Contacts
                    </h3>
                  </div>
                </Link>
              </Col>,
              <Col
                bsClass="col"
                componentClass="div"
                sm={6}
              >
                <Link
                  className="navLink"
                  replace={false}
                  to="/listings"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Listings
                    </h3>
                  </div>
                </Link>
              </Col>,
              <Col
                bsClass="col"
                componentClass="div"
                sm={6}
              >
                <Link
                  className="navLink"
                  replace={false}
                  to="/emails"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Emails
                    </h3>
                  </div>
                </Link>
              </Col>,
              <Col
                bsClass="col"
                componentClass="div"
                sm={6}
              >
                <Link
                  className="navLink"
                  replace={false}
                  to="/profile"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Profile
                    </h3>
                  </div>
                </Link>
              </Col>,
            ],
            "componentClass": "div",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "col",
                "children": <Link
                  className="navLink"
                  replace={false}
                  to="/contacts"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Contacts
                    </h3>
                  </div>
                </Link>,
                "componentClass": "div",
                "sm": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Contacts
                    </h3>
                  </div>,
                  "className": "navLink",
                  "replace": false,
                  "to": "/contacts",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <h3
                      className="dashText"
                    >
                      Contacts
                    </h3>,
                    "className": "dashCard",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Contacts",
                      "className": "dashText",
                    },
                    "ref": null,
                    "rendered": "Contacts",
                    "type": "h3",
                  },
                  "type": "div",
                },
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "col",
                "children": <Link
                  className="navLink"
                  replace={false}
                  to="/listings"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Listings
                    </h3>
                  </div>
                </Link>,
                "componentClass": "div",
                "sm": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Listings
                    </h3>
                  </div>,
                  "className": "navLink",
                  "replace": false,
                  "to": "/listings",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <h3
                      className="dashText"
                    >
                      Listings
                    </h3>,
                    "className": "dashCard",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Listings",
                      "className": "dashText",
                    },
                    "ref": null,
                    "rendered": "Listings",
                    "type": "h3",
                  },
                  "type": "div",
                },
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "col",
                "children": <Link
                  className="navLink"
                  replace={false}
                  to="/emails"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Emails
                    </h3>
                  </div>
                </Link>,
                "componentClass": "div",
                "sm": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Emails
                    </h3>
                  </div>,
                  "className": "navLink",
                  "replace": false,
                  "to": "/emails",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <h3
                      className="dashText"
                    >
                      Emails
                    </h3>,
                    "className": "dashCard",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Emails",
                      "className": "dashText",
                    },
                    "ref": null,
                    "rendered": "Emails",
                    "type": "h3",
                  },
                  "type": "div",
                },
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "col",
                "children": <Link
                  className="navLink"
                  replace={false}
                  to="/profile"
                >
                  <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Profile
                    </h3>
                  </div>
                </Link>,
                "componentClass": "div",
                "sm": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <div
                    className="dashCard"
                  >
                    <h3
                      className="dashText"
                    >
                      Profile
                    </h3>
                  </div>,
                  "className": "navLink",
                  "replace": false,
                  "to": "/profile",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <h3
                      className="dashText"
                    >
                      Profile
                    </h3>,
                    "className": "dashCard",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Profile",
                      "className": "dashText",
                    },
                    "ref": null,
                    "rendered": "Profile",
                    "type": "h3",
                  },
                  "type": "div",
                },
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
